behavior-modules/help.js

```diff
@@ -0,0 +1,33 @@
+const Behavior = require('../models/behavior');
+
+function helpMessage (name, service) {
```

Comment:
@jywarren this action will be called (infact all such kind of actions will be called when):

1. The trigger (`message` here) is triggered.
2. The keyword (`help` here) is mentioned inside the message.